package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "postinstall": "gulp"
+    "postinstall": "gulp",
+    "prettier": "prettier --write 'frontend/app/**/*.js' && prettier --write 'frontend/**/*.less'"
```

Comment:
like this, made a bash script on master but this is better

Reply:
Oops! Didn't see the script.